tests/plugins/emoji/basic.js

```diff
@@ -0,0 +1,71 @@
+/* bender-tags: emoji */
+/* bender-ckeditor-plugins: emoji,toolbar,stylescombo,format,clipboard */
+
+( function() {
+	'use strict';
+
+
+	bender.editors = {
+		inline: {
+			creator: 'inline',
+			name: 'inline',
+			startupData: '<p>foo:grinning_face:bar</p>'
+		},
+		classic: {
+			creator: 'replace',
+			name: 'classic',
+			startupData: '<p>foo:grinning_face:bar</p>'
+		}
+	};
+
+	var tests = {
+		setUp: function() {
+			if ( CKEDITOR.env.ie && CKEDITOR.env.version < 9 ) {
+				assert.ignore();
+			}
+		},
+		'test emoji is converted during editor creation': function( editor ) {
+			assert.areSame( '<p>foo😀bar</p>', editor.getData() );
+		},
+		'test emoji is converted during setData': function( editor ) {
+			editor.setData( '<p>bar:slightly_smiling_face:baz</p>', {
+				callback: function() {
+					resume( function() {
+						assert.areSame( '<p>bar🙂baz</p>', editor.getData() );
+					} );
+				}
+			} );
+			wait();
+		},
+		'test emoji is converted durign insertText': function( editor, bot ) {
```

Comment:
This test is good, I'd add a complementary test that ensures that in case of unmatched emoji identifier the text is being inserted. So try to insert `:ck_nonexisting_emoji:`. And ensure that it was inserted as text.